### PR TITLE
Feat/global navigation

### PR DESCRIPTION
**Pull request by 0xRowdy**
Created and styled global navigation component, created Route for My Actions

Screenshot 2024-02-05 at 1 40 27 PM
Screenshot 2024-02-05 at 1 40 33 PM
Screenshot 2024-02-05 at 1 54 37 PM

### CODE REVIEW

File: app/config/site.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		twitter: string;
+		github: string;
+	};
+}
+
+export const siteConfig: SiteConfig = {
+	name: "VoiceDeck",
+	title: "VoiceDeck - From Individual Actions to Collective Impact",
+	description:
+		"We enable journalists to effect real change by bringing critical stories to light. Your contributions directly support this mission, sustaining  journalism and bolstering investigative reporting that matters.",
```
Comment (CJ-Rose) [inline app/config/site.ts:17]: if you remove "Voicedeck - " from the title and change to sentence case, we can pull this in to `_index.tsx` for the hero content. also there is an extra space after "sustaining".
is it meant to be used somewhere else?

**Review by CJ-Rose — COMMENTED** [review]
re: .DS_Store file:
https://github.com/VoiceDeck/app/pull/13#discussion_r1467747193

did not get added to .gitignore 🙃

File: app/config/site.ts (hunk 1 above)
Comment (0xRowdy) [reply app/config/site.ts:17]: Good catch.
I was thinking we could use the config to populate the metadata for the SEO also.

Comment (0xRowdy) [reply app/config/site.ts]: done

**Pull request merged**